commands-yml/commands/mobile-command.yml

```diff
@@ -93,6 +93,7 @@
       | mobile:scrollBackTo | Scroll from one element to another | `{elementId, elementToId}` | `{elementId: 2, elementToId: 1}` |
       | mobile:scroll | Scroll the given scrollable element until the element identifier by `strategy` and `selector` becomes visible since Appium 1.18.0. An optional `maxSwipes` property can limit the number of scroll actions. It accepts an optional `element` property which should be the element identifier and scrollable. Without this property, Appium will select the first currently available scrollable container. Please read `mobileScroll` section in [appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver/blob/master/lib/commands/general.js) for more details. | `{element, strategy, selector, maxSwipes}` | `{strategy: 'accessibility id', selector: 'target content description'}` |
       | mobile:viewportScreenshot | Like [screenshot](/commands/session/screenshot/) but only includes contents of viewport | <none> | <none> |
+      | mobile:viewportRect | Returns the coordinates and size of the viewports since Appium 1.18.0. | <none> | <none> |
```

Comment:
Thanks, one more.
Could you leave a link to https://github.com/appium/appium-uiautomator2-driver/blob/master/lib/commands/general.js to be able to check the return value format as the docstring.

Reply:
I don't quite understand the request. What exactly do you want me to do? Where should I add a link? Where should the link point? Is there another place I can see example for such implementation already?

Reply:
Add the link like https://github.com/appium/appium/pull/14450/files#diff-29516863f13863f4f028f30051fe44a2R98 and https://github.com/appium/appium/pull/14450/files#diff-29516863f13863f4f028f30051fe44a2R94 so that readers can take a look at details such as response format. Line number may change so a link to the file is fine.